### PR TITLE
long range buckshot

### PR DESCRIPTION
**Pull request by Dorodomki**
## About The Pull Request
makes the buckshot become useful at long and close range, maybe losing his PB effect but becoming ideal to kill roaches, injure enemies without armor and making it a more useful ammo

## Changelog
:cl:
tweak: makes the buckshot be long range ammunition with 7 pellets equal to a weak 9mm round, mostly a test 
/:cl:

### CODE REVIEW

File: code/modules/projectiles/projectile/bullettypes.dm [hunk 1]
```diff
@@ -893,10 +893,18 @@
 	recoil = 6
 
 /obj/item/projectile/bullet/pellet/shotgun/scrap
-	damage_types = list(BRUTE = 7.5)
-	affective_damage_range = 4
-	affective_ap_range = 4
-	recoil = 4
+		damage_types = list(BRUTE = 4)
```
Comment (Trilbyspaceclone) [inline code/modules/projectiles/projectile/bullettypes.dm:896]:
```suggestion
	damage_types = list(BRUTE = 4)
```

**Comment by Trilbyspaceclone** [comment]
I see no issue with this

File: code/modules/projectiles/projectile/bullettypes.dm [hunk 2]
```diff
@@ -893,10 +893,18 @@
 	recoil = 6
 
 /obj/item/projectile/bullet/pellet/shotgun/scrap
-	damage_types = list(BRUTE = 7.5)
-	affective_damage_range = 4
-	affective_ap_range = 4
-	recoil = 4
+	damage_types = list(BRUTE = 4)
+	base_spread = 90
+	range_step = 8		//projectile will lose a fragment each time it travels this distance.
```
Comment (Trilbyspaceclone) [inline code/modules/projectiles/projectile/bullettypes.dm:898]: im willing to let this be 2

File: code/modules/projectiles/projectile/bullettypes.dm [hunk 3]
```diff
@@ -987,15 +995,17 @@
 /obj/item/projectile/bullet/pellet/shotgun
 	name = "shrapnel"
 	icon_state = "birdshot-1"
-	damage_types = list(BRUTE = 11)
-	agony = 5
-	pellets = 4
+	damage_types = list(BRUTE = 6.5)
+	base_spread = 90
+	range_step = 8		//projectile will lose a fragment each time it travels this distance.
```
Comment (Trilbyspaceclone) [inline code/modules/projectiles/projectile/bullettypes.dm:1000]: this should be like 2 not 8

**Comment by ThePainkiller** [comment]
Oh boy, cultists with shotguns will be DEADLIER now! Joy!

File: code/modules/projectiles/projectile/bullettypes.dm (hunk 2 above)
Comment (Trilbyspaceclone) [inline code/modules/projectiles/projectile/bullettypes.dm:898]:
```suggestion
	range_step = 2		//projectile will lose a fragment each time it travels this distance.
```

File: code/modules/projectiles/projectile/bullettypes.dm (hunk 3 above)
Comment (Trilbyspaceclone) [inline code/modules/projectiles/projectile/bullettypes.dm:1000]:
```suggestion
	range_step = 2		//projectile will lose a fragment each time it travels this distance.
```

**Pull request merged**